org.geeqie.Geeqie.json

```diff
@@ -0,0 +1,134 @@
+{
+    "app-id": "org.geeqie.Geeqie",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "geeqie",
+    "rename-desktop-file": "geeqie.desktop",
+    "rename-icon": "geeqie",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host"
```

Comment:
Any reason why it requires host access?

Reply:
Without it you can open single images, but you can't move to the next one inside the same folder

Reply:
Can it be limited to xdg-pictures instead? It's easy to overwrite for users, but we want the defaults to be secure as much as possible for given app.

Reply:
I understand what you mean, but easy to overwrite is not completely true in my opinion. I'd have to look for the desktop files and change to correct line. How many people would know that this is the right way to do it and be willing to do that? 

I think that if they can't open pictures outside of Pictures, they will stop using the geeqie flatpak (or even flatpaks in general) and move to "normal" apps instead which offer no sandboxing at all.

Reply:
At some point there will hopefully a way to (implicitly) ask the user to grant access, but until then should the app only work at 90% percent of all use cases?

Reply:
But I'll limit it if you still want me to

Reply:
Other image viewers also have host access. I think its just a reality of the application since no portal exists to deal with directories.

Reply:
Hm, you're right. For some reason I thought EOG doesn't. Marking as resolved then.

Reply:
Hm, you're right. For some reason I thought EOG doesn't. Marking as resolved then.